src/Grb.Building.Api/Handlers/GetAllJobRecordsHandler.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace Grb.Building.Api.Handlers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Infrastructure;
+    using Infrastructure.Query;
+    using MediatR;
+    using Microsoft.EntityFrameworkCore;
+    using TicketingService.Abstractions;
+
+    public record GetAllJobRecordsRequest(
+        Pagination Pagination,
+        List<JobRecordStatus> JobRecordStatuses,
+        Guid JobId) : IRequest<GetAllJobRecordsResponse>;
+
+    public record GetAllJobRecordsResponse(object[] JobRecords, Uri NextPage);
```

Comment:
I'm not the biggest fan of calling it Get**All**JobRecords or Get**AllJobs** because we are filtering on job id (or other filters). Maybe just GetJobs and GetJobRecords?

Reply:
And why are we returing JobRecords as object[] ?